### ISSUE TITLE
VintageStoryServer.exe server-side crash

### ISSUE THREAD

**Issue by Shuro** [anegostudios/VintageStory-Issues#952]
**Game Version:** 1.15.0-pre2
**Platform:** Linux (Alpine Container)
**Modded:** No

### Description
Server crashed 2 times in the last days and resetted the progress for about half a day.

### How to reproduce
I think I destroyed a stone cluster for collecting rocks, but no idea.

### Logs
```
Version: v1.15.0-pre.2 (Unstable)05/31/2021 18:13:59: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Common.WorldChunk.GetLocalBlockAtBlockPos (Vintagestory.API.Common.IWorldAccessor world, Vintagestory.API.MathTools.BlockPos position) [0x0003a] in <d63bf8eb56554bb09eb9e19878a3a877>:0
  at Vintagestory.GameContent.WeatherSimulationSnowAccum.UpdateSnowLayer (Vintagestory.GameContent.SnowAccumSnapshot sumsnapshot, System.Boolean ignoreOldAccum, Vintagestory.API.Common.IServerMapChunk mc, Vintagestory.API.MathTools.Vec2i chunkPos, Vintagestory.API.Common.IWorldChunk[] chunksCol) [0x00326] in <29c94d3afb78469eb11b522771548918>:0
  at Vintagestory.GameContent.WeatherSimulationSnowAccum.GetSnowUpdate (Vintagestory.GameContent.WeatherSimulationRegion simregion, Vintagestory.API.Common.IServerMapChunk mc, Vintagestory.API.MathTools.Vec2i chunkPos, Vintagestory.API.Common.IWorldChunk[] chunksCol) [0x00173] in <29c94d3afb78469eb11b522771548918>:0
  at Vintagestory.GameContent.WeatherSimulationSnowAccum.UpdateSnowLayerOffThread (Vintagestory.GameContent.WeatherSimulationRegion simregion, Vintagestory.API.Common.IServerMapChunk mc, Vintagestory.API.MathTools.Vec2i chunkPos) [0x0006b] in <29c94d3afb78469eb11b522771548918>:0
  at Vintagestory.GameContent.WeatherSimulationSnowAccum.onThreadTick () [0x0009f] in <29c94d3afb78469eb11b522771548918>:0
  at System.Threading.ThreadHelper.ThreadStart_Context (System.Object state) [0x00014] in <d13c8b563008422a8c5aaec0a74089cc>:0
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in <d13c8b563008422a8c5aaec0a74089cc>:0
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <d13c8b563008422a8c5aaec0a74089cc>:0
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state) [0x0002b] in <d13c8b563008422a8c5aaec0a74089cc>:0
  at System.Threading.ThreadHelper.ThreadStart () [0x00008] in <d13c8b563008422a8c5aaec0a74089cc>:0
-------------------------------
Version: v1.15.0-pre.2 (Unstable)06/02/2021 17:17:38: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Common.WorldChunk.GetLocalBlockAtBlockPos (Vintagestory.API.Common.IWorldAccessor world, Vintagestory.API.MathTools.BlockPos position) [0x0003a] in <d63bf8eb56554bb09eb9e19878a3a877>:0
  at Vintagestory.GameContent.WeatherSimulationSnowAccum.UpdateSnowLayer (Vintagestory.GameContent.SnowAccumSnapshot sumsnapshot, System.Boolean ignoreOldAccum, Vintagestory.API.Common.IServerMapChunk mc, Vintagestory.API.MathTools.Vec2i chunkPos, Vintagestory.API.Common.IWorldChunk[] chunksCol) [0x001af] in <29c94d3afb78469eb11b522771548918>:0
  at Vintagestory.GameContent.WeatherSimulationSnowAccum.GetSnowUpdate (Vintagestory.GameContent.WeatherSimulationRegion simregion, Vintagestory.API.Common.IServerMapChunk mc, Vintagestory.API.MathTools.Vec2i chunkPos, Vintagestory.API.Common.IWorldChunk[] chunksCol) [0x00173] in <29c94d3afb78469eb11b522771548918>:0
  at Vintagestory.GameContent.WeatherSimulationSnowAccum.UpdateSnowLayerOffThread (Vintagestory.GameContent.WeatherSimulationRegion simregion, Vintagestory.API.Common.IServerMapChunk mc, Vintagestory.API.MathTools.Vec2i chunkPos) [0x0006b] in <29c94d3afb78469eb11b522771548918>:0
  at Vintagestory.GameContent.WeatherSimulationSnowAccum.onThreadTick () [0x0009f] in <29c94d3afb78469eb11b522771548918>:0
  at System.Threading.ThreadHelper.ThreadStart_Context (System.Object state) [0x00014] in <d13c8b563008422a8c5aaec0a74089cc>:0
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in <d13c8b563008422a8c5aaec0a74089cc>:0
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <d13c8b563008422a8c5aaec0a74089cc>:0
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state) [0x0002b] in <d13c8b563008422a8c5aaec0a74089cc>:0
  at System.Threading.ThreadHelper.ThreadStart () [0x00008] in <d13c8b563008422a8c5aaec0a74089cc>:0
-------------------------------
client-crash.txt (END)
```

**Comment by Craluminum2413** [comment]
#908 ?

**Comment by copygirl** [comment]
Indeed, this is a duplicate. Thanks for reporting, though!

**Comment by radfast** [comment]
I believe this is fixed in `1.15-pre.3` (see #News channel on Discord).  If you still have the same issue please re-report it and we should give it high priority.
